modmail/__init__.py

```diff
@@ -38,14 +42,15 @@
 
 file_handler.setLevel(logging.TRACE)
 
-coloredlogs.install(
-    level=LOG_LEVEL,
-    fmt=fmt,
-    datefmt=datefmt,
-)
+# configure trace color
```

Comment:
Couldn't put this anywhere else but rather than the relational path of `log_file` wouldn't be better to use `Path("logs", "bot.log")`